File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/UnknownRollCallException.java
package com.github.dedis.popstellar.utility.error;

import com.github.dedis.popstellar.R;

public class UnknownRollCallException extends UnknownEventException {

  public UnknownRollCallException(String id) {
    super("Roll call", id);
  }

  @Override
  public int getUserMessage() {
    return R.string.unknown_roll_call_exception;
  }

  @Override
  public Object[] getUserMessageArguments() {
    return new Object[0];
  }
}

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/UnknownRollCallException.kt
package com.github.dedis.popstellar.utility.error

import com.github.dedis.popstellar.R

class UnknownRollCallException(id: String) : UnknownEventException("Roll call", id) {
  override val userMessage: Int
    get() = R.string.unknown_roll_call_exception

  override val userMessageArguments: Array<Any?>
    get() = arrayOfNulls(0)
}
